python/CMakeLists.txt

```diff
@@ -16,34 +19,48 @@ else()
   target_link_libraries(pyf3d PRIVATE libf3d)
 endif()
 
+set(f3d_python_package_name "f3d")
+set(f3d_python_package_suffix "python${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}/site-packages")
+set(f3d_python_lib_output "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/${f3d_python_package_suffix}/${f3d_python_package_name}")
+
 set_target_properties(pyf3d PROPERTIES
   CXX_STANDARD 14
   CXX_VISIBILITY_PRESET hidden
-  LIBRARY_OUTPUT_NAME "f3d"
+  LIBRARY_OUTPUT_DIRECTORY "${f3d_python_lib_output}"
   )
 
-if(WIN32)
-  # On Windows, the python module needs to be in the same folder than f3d.dll
-  # Usage of PATH to find the DLL is not possible, see https://stackoverflow.com/a/64303856/2609654
-  set_target_properties(pyf3d PROPERTIES
-    LIBRARY_OUTPUT_DIRECTORY "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}"
-    )
+if (APPLE)
+  set_target_properties(pyf3d PROPERTIES INSTALL_RPATH "@loader_path/../../../${CMAKE_INSTALL_LIBDIR}")
+elseif (UNIX)
+  set_target_properties(pyf3d PROPERTIES INSTALL_RPATH "$ORIGIN/../../../${CMAKE_INSTALL_LIBDIR}")
```

Comment:
could you use target API instead ?

Reply:
What is target api?

Reply:
On the phone right now but something like that?

`get_target_property(FOO_SOURCE_DIR Foo SOURCE_DIR)`

Reply:
I still don't get it. What target do you want to use?  
We are installing the python module in `<prefix>/lib/pythonXXX/site-packages/f3d`, we just want to tell the module to look in `<prefix>/lib`

Reply:
I'm suggesting to not use a relative path to find the right location if possible.

Reply:
but I completely misread what this was. nevermind.